etc/avocado/avocado.conf

```diff
@@ -80,3 +80,10 @@ skip_broken_plugin_notification = []
 # options or test types).
 # The keyword "@DEFAULT" will be replaced with all available unused loaders.
 loaders = ['file', '@DEFAULT']
+
+[plugins.job.mail]
+configured = yes
+recipient = dmon@mct2.qa.sw.ru
```

Comment:
I don't think you intended to include this one... Anyway I'd suggest using `avocado.d/mail.conf` for details regarding plugins :wink: